### ISSUE TITLE
[29126][1][FATAL] Chart class not found.

### ISSUE THREAD

**Issue by greenlanegreb** [salesagility/SuiteCRM#7748]
Done

 Nothing

 I don't believe it is. 

#### Issue
[29126][1][FATAL] Chart class not found is appearing in the suitecrm.log. I have searched online and came across some proposed fixes but I don't know if they are agreed by the Developers. Also, the chap has not received a reply - would somebody be kind enough to provide an authoritative reply on his thread please and categorize the bug accordingly https://suitecrm.com/suitecrm/forum/installation-upgrade-help/27041-fresh-install-of-suitecrm-version-7-11-7-dashlet-not-working

Many thanks. 

#### Expected Behavior
 Error should not appear, dashboard should work. 

#### Actual Behavior
 Dashboard Does not Work, Error recorded in suitecrm.log

#### Possible Fix
 Not a coder i'm afraid!

#### Steps to Reproduce

1. Please see suitecrm.log
2.
3.
4.

#### Context
 I'm trying to get a clean log file!

 I'll leave that for you, in your wisdom to decide. 

#### Your Environment

* SuiteCRM Version used: 7.11.7

* Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): N/A
* Environment name and version (e.g. MySQL, PHP 7): Maria DB 10.3.15 PHP 7.0.33
* Operating System and version (e.g Ubuntu 16.04): Debian Buster with Apache 2.4.38

**Comment by pgorod** [comment]
I think this is a duplicate of #7696

There is a fix which should be in 7.11.8.

**Comment by samus-aran** [comment]
Hi there we are closing this since it has been included in 7.11.8. If it is still an issue in the most recent version then please re-open this issue.

**Comment by greenlanegreb** [comment]
Thanks Ashley 

I was waiting for somebody who knows better than me to confirm as it appeared the other chap wasn’t quite sure. 

Kind Regards 😊

**Comment by marcovansteen** [comment]
In my fresh install of 7.11.8 this issue appeared after using the software for a couple of days. 
I think the issue should be re-opened. Can somebody do this? I don't know how to re-open this issue.

**Comment by iba-jsykes** [comment]
This happened to me in the following version after upgrading : 
Version 7.8.31
Sugar Version 6.5.25 (Build 344)

Copying the missing folder over and clearing cache in browser / include folder has no effect.

**Comment by Dillon-Brown** [comment]
Hi @marcovansteen, I've left a comment on the original issue here: #7696. Thanks.

**Comment by marcovansteen** [comment]
OK, thank you.

Op ma 16 sep. 2019 om 12:01 schreef Dillon Brown <notifications@github.com>:

**Comment by mikebakke** [comment]
Out of (possible) interest. I noticed that message in my log today so found this issue here while searching. To fix I had to use the 7.11.x to 7.11.8 upgrade archive and copy from there the complete Jit directory. Since then I have not had any error messages.

Also interesting is that I have not (so far) encountered the intermittent hangs which used to require a quick F5 and also none of the apache errors related to being unable to stat cache/index.php which is logged as a separate issue. Early days but the system seems much more "snappy" overall after adding Jit
